Foundation/FileManager.swift

```diff
@@ -802,7 +802,7 @@ open class FileManager : NSObject {
     }
 
 #if os(Windows)
-    private func joinPath(prefix: String, suffix: String) -> String {
+    fileprivate func joinPath(prefix: String, suffix: String) -> String {
```

Comment:
mmm, pretty sure that we want this to remain `private` as `fileprivate` is discouraged now I believe.  There is no need for this to be a member function, just extract it to a standalone function and let it remain `private` or make it `internal`.

Reply:
Sure, moved it to an internal func.